lang/python/python3-speedtest-cli/Makefile

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (C) 2019 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
```

Comment:
May I ask you where did you get this from? I think that you used some Makefile as inspiration, which is good, but you should omit this or assign it to you. See https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md

Anyway, if CZ.NIC, who manufactures Turris routers has this in their repository, it would be upstreamed. You can trust me on this one. ;)

Reply:
:) Thanks. Yes, I referred to other Makefile as a reference, As per the contribution guide I cleared that now.